inspirehep/modules/references/processors.py

```diff
@@ -180,9 +180,9 @@ def add_refextract_authors_str(self, authors_str):
         migration.
         """
         for author in _split_refextract_authors_str(authors_str):
```

Comment:
well, the function does what it says. But, since it is the only place where is used, I am going to refactor it and put the 'cleaning' code in it.